login-fire-social.html

```diff
@@ -158,36 +156,34 @@
       },
 
       /**
-       * GitHub scopes.
+       * The scopes that you want to access from GitHub sign in. Available
+       * scopes at https://developer.github.com/v3/oauth/#scopes
        *
        * @type {String}
-       * @default ''
        */
       githubScopes: {
-        type: String,
-        value: ''
```

Comment:
Default values for scopes were removed to avoid trigger uselessly the observer `_scopesChanged`.